test/generator/exercise_test.rb

```diff
@@ -39,21 +39,65 @@ def test_update_example_solution
       mock_file.verify
     end
 
+    def test_create_tests_file
```

Comment:
Ok it's back, good. Still think the rename is preferable.
Also leads to quieter diffs.